sys/include/net/gnrc/tcp/config.h

```diff
@@ -54,22 +79,22 @@ extern "C" {
 /**
  * @brief MSS Multiplicator = Number of MSS sized packets stored in receive buffer
  */
-#ifndef GNRC_TCP_MSS_MULTIPLICATOR
-#define GNRC_TCP_MSS_MULTIPLICATOR (1U)
+#ifndef CONFIG_GNRC_TCP_MSS_MULTIPLICATOR
+#define CONFIG_GNRC_TCP_MSS_MULTIPLICATOR (1U)
 #endif
 
 /**
  * @brief Default receive window size
  */
 #ifndef GNRC_TCP_DEFAULT_WINDOW
-#define GNRC_TCP_DEFAULT_WINDOW (GNRC_TCP_MSS * GNRC_TCP_MSS_MULTIPLICATOR)
+#define GNRC_TCP_DEFAULT_WINDOW (GNRC_TCP_MSS * CONFIG_GNRC_TCP_MSS_MULTIPLICATOR)
 #endif
 
```

Comment:
Why is the default receive window size not configurable?

Reply:
The value is now found from two configurations. Is there a need to allow the macro to be configured directly?

Reply:
This value marks the maximum amount of bytes that can be received from the peer at a given moment.
In general it makes sence to elide this value from "n times maximum segment size" however this parameter influences TCPs behavior significantly. Making in configurable might have its benefits.